content/tutorials/importing-azure-infrastructure/index.md

```diff
@@ -0,0 +1,406 @@
+            "blob": {
+                "enabled": True,
+                "key_type": azure_native.storage.KeyType.ACCOUNT,
+            },
+            "file": {
+                "enabled": True,
+                "key_type": azure_native.storage.KeyType.ACCOUNT,
+            },
+        },
+    },
+    kind=azure_native.storage.Kind.STORAGE_V2,
+    large_file_shares_state=azure_native.storage.LargeFileSharesState.ENABLED,
+    location="northcentralus",
+    minimum_tls_version=azure_native.storage.MinimumTlsVersion.TLS1_2,
+    network_rule_set={
+        "bypass": azure_native.storage.Bypass.AZURE_SERVICES,
+        "default_action": azure_native.storage.DefaultAction.ALLOW,
+    },
+    public_network_access=azure_native.storage.PublicNetworkAccess.ENABLED,
+    resource_group_name="pulumi-tutorials",
+    sku={
+        "name": azure_native.storage.SkuName.STANDARD_RAGRS,
+    },
+    opts = pulumi.ResourceOptions(protect=True))
+```
+
+Next, copy the emitted code snippet and replace the contents of your Pulumi program file with it. Then, save the file and run `pulumi up`. You should see that the update produces no changes:
+
+```bash
+$ pulumi up
+
+Previewing update (dev)
+
+     Type                 Name                     Plan
+     pulumi:pulumi:Stack  dev
+
+Resources:
+    2 unchanged
+
+Do you want to perform this update? yes
+
+Updating (dev)
+
+     Type                 Name                     Status
+     pulumi:pulumi:Stack  dev
+
+Resources:
+    2 unchanged
+
+Duration: 2s
+```
+
+The resource is now under management with Pulumi!
+
+{{< notes type="info" >}}
+
+Resources imported with the CLI are marked as __protected__ to guard against accidental deletion. If, for example, you forgot to append the generated code to your program before running another `pulumi up`, Pulumi would first interpret the missing code as an intention to delete the resource. The `protect` property will prevent this from happening, leaving the resource intact. If you ever want to delete this resource, you will have to set the `protect` property to `false` in the code. You can learn more by visiting the [Resource option: protect documentation](/docs/concepts/options/protect/).
+
+{{< /notes >}}
+
+### Import in bulk
+
+The `pulumi import` command also enables you to import resources in bulk for scenarios in which you need to bring multiple resources under management with Pulumi. To do so, you will need to create a JSON file that has all of the required information for each resource: a `type`, a desired `name`, and an `id`.
+
+To start, return to the Azure console and create two additional Storage Accounts. Next, return to your program folder and create a new file called `resources.json`. Inside of this file, copy and paste the following JSON object, making sure to replace the values of the `id` parameters with the actual values for your environment:
+
+```json
+{
+    "resources": [
+        {
+            "type": "aws:s3/bucket:Bucket",
```

Comment:
I think this JSON file needs to be updated. It seems to be copy pasta from the AWS tutorial (wrong resource type, should be `azure-native:storage:StorageAccount`).

Reply:
Good catch, thank you!